build.gradle

```diff
@@ -176,6 +176,8 @@ dependencies {
   compile group: 'org.springframework.boot', name: 'spring-boot-starter-json'
   compile group: 'uk.gov.hmcts.reform', name: 'health-spring-boot-starter', version: '0.0.4'
 
+  compile group: 'javax.xml.bind', name: 'jaxb-api', version: '2.3.1' // Needed for openjdk 11 to execute test
```

Comment:
test? or it's app boot test kind of thing?

Reply:
needed for both unit and int tests

Reply:
right. will have to investigate especially if it's not essential for the app itself. inclusion just because something is incompatible is problem with dependencies. nvm, carry on :D